sw-precache-config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
I had to add it because it wasn't picking up `.pngs`. It's also needed later for the raven stuff.